Readme.md

```diff
@@ -46,15 +46,90 @@ If you'd like to build your own VFS for Git Windows installer:
 * Create a folder to clone into, e.g. `C:\Repos\VFSForGit`
 * Clone this repo into the `src` subfolder, e.g. `C:\Repos\VFSForGit\src`
 * Run `\src\Scripts\BuildGVFSForWindows.bat`
-* You can also build in Visual Studio by opening `src\GVFS.sln` (do not upgrade any projects) and building. However, the very first 
+* You can also build in Visual Studio by opening `src\GVFS.sln` (do not upgrade any projects) and building. However, the very first
 build will fail, and the second and subsequent builds will succeed. This is because the build requires a prebuild code generation step.
 For details, see the build script in the previous step.
 
 The installer can now be found at `C:\Repos\VFSForGit\BuildOutput\GVFS.Installer\bin\x64\[Debug|Release]\SetupGVFS.<version>.exe`
 
+## Building VFS for Git on Mac
+
+VFS for macOS is still in progress. You can build it, but this will not create a macOS VFS installer the same way the current Windows build will.
+
+* Ensure you have Xcode installed and have accepted the terms of use and launched Xcode at least once.
+
+* Disable "System Integrity Protection" (for loading unsigned kexts) by booting into recovery mode (`[Win/⌘] + R` while booting).
+  Once booted into recovery mode open Utilities -> Terminal to launch a termnial. Enter:
+  ```
+  csrutil disable
+  ```
+  to disable SIP.
+
+* Install [Visual Studio for Mac ](https://visualstudio.microsoft.com/vs/mac). (This will also install the `dotnet` CLI).
+
+* If you still do not have the `dotnet` cli `>= v2.1.300` installed [manually install it](https://www.microsoft.com/net/download/dotnet-core/2.1)
+
+* If you are not currently an Apple Developer you will need to become one so that you can either use Microsoft certs or create your own certs for signing purposes. (If you are a Microsoft employee you must use your alias@microsoft.com when creating your Apple account.)

```

Comment:
I would remove all of this and instead point to a link for managing certs in Xcode. There's no reason for anyone to use Microsoft certs, nor do we want to open that up. For local development, people just need to generate their own cert and sign their kext, and that is all.

Reply:
How is [this](https://developer.apple.com/documentation/security/certificate_key_and_trust_services/certificates) as the resource to link to?

Reply:
That page seems too high level on what a certificate is (we should point them to codesigning resources using certificates), this might be a better starting point https://developer.apple.com/support/code-signing/ 

In general, Xcode should be able to do all the right things for them once they're signed in.